<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Abhijit" failures="1" tests="4" name="Default test" time="14.212" errors="0" timestamp="2024-09-05T19:07:54 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.648"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.953">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepsDefination.LoginStepsDefination.user_should_navigate_to_the_home_page(LoginStepsDefination.java:42)
at âœ½.User should navigate to the home  page(file:///C:/Users/abhij/eclipse-workspace/BDDFramework/src/test/resources/features/login.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.987"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="7.144"/>
</testsuite> <!-- Default test -->
